app/controllers/providers/addresses_controller.rb

```diff
@@ -15,14 +15,16 @@ def create
       if @form.save
         redirect_to action_for_next_step(options: { application: applicant.legal_aid_application })
       else
+        @lookup_postcode = params[:address][:lookup_postcode]
```

Comment:
If `:lookup_postcode` and `:lookup_error` are already being submitted through the form, 

`@form = Applicants::AddressForm.new(form_params)`

is already getting old of those values as it's initialised with them (`form_params`), so when rendering the view, there's no need to have these instance variables declared (they're not being used, and the form object already has the values for them anyway.